encoder/vaapiencoder_base.cpp

```diff
@@ -89,12 +89,12 @@ YamiStatus VaapiEncoderBase::start(void)
 void VaapiEncoderBase::flush(void)
 {
     AutoLock l(m_lock);
-    m_output.clear();
 }
 
 YamiStatus VaapiEncoderBase::stop(void)
 {
     FUNC_ENTER();
+    m_output.clear();
     cleanupVA();
```

Comment:
@chivakker , this change it not only for h264 encoder. It is relative how to use stop() and flush() interface defined in VideoEncoderInterface.cpp. The right pipeline is : start()->encode()->flush()->getoutput()->stop(). If we clear m_output, we can not get output data when we call getoutput().